### PR TITLE
[FrameworkBundle] cache:pool:delete command

### LINKED ISSUE #9454: [FrameworkBundle] Add command to delete an item from a cache pool
The command is called `cache:pool:delete` See https://github.com/symfony/symfony/pull/26223

### PR DESCRIPTION
**Pull request by snoek09**
This fixes #9454.

### CODE REVIEW

File: components/cache/cache_pools.rst [hunk 1]
```diff
@@ -140,6 +151,10 @@ when all items are successfully deleted)::
         # clears the "cache.validation" and "cache.app" pool
         $ php bin/console cache:pool:clear cache.validation cache.app
 
+.. versionadded:: 4.1
+
+The ``cache:pool:delete`` command was introduced in Symfony 4.1.
```
Comment (xabbuh) [inline components/cache/cache_pools.rst]: This sentence needs to be indented by four spaces.

File: components/cache/cache_pools.rst [hunk 2]
```diff
@@ -140,6 +151,10 @@ when all items are successfully deleted)::
         # clears the "cache.validation" and "cache.app" pool
         $ php bin/console cache:pool:clear cache.validation cache.app
 
+.. versionadded:: 4.1
+
```
Comment (xabbuh) [inline components/cache/cache_pools.rst]: the blank line must be removed

**Review by xabbuh — APPROVED** [review]

File: components/cache/cache_pools.rst [hunk 3]
```diff
@@ -127,9 +127,20 @@ when all items are successfully deleted)::
...
     If the cache component is used inside a Symfony application, you can remove
-    *all items* from the *given pool(s)* using the following command (which resides within
+    items from cache pools using the following commands (which reside within
     the :ref:`framework bundle <framework-bundle-configuration>`):
 
+    To remove *one specific item* from the *given pool*:
+
+    .. code-block:: terminal
+
+        $ php bin/console cache:pool:delete <cache-pool-name> <cache-key-name>
```
Comment (theofidry) [inline components/cache/cache_pools.rst]: one thing that is not clear for me here is how you get the list of cache pool names :/

Comment (xabbuh) [reply components/cache/cache_pools.rst]: You configured them yourself in the `framework.cache` section. Do you think we need to clarify that?

Comment (theofidry) [reply components/cache/cache_pools.rst]: I think so. Isn't there `app` and `system` as well by default?

**Comment by javiereguiluz** [comment]
Thanks for this contribution! We've merged it and opened #9782 to not forget about the issue mentioned by @theofidry

**Pull request merged**